pkgs/applications/misc/colorstorm/default.nix

```
@@ -0,0 +1,18 @@
+{ lib, fetchFromGitHub, zigPackages }:
+zigPackages.buildZigPackage {
+  name = "colorstorm";
+  src = fetchFromGitHub
+    {
+      repo = "colorstorm";
+      owner = "benbusby";
+      rev = "3a05f395da5e72d0d041268cd532ebd22e949cf6";
+      sha256 = "sha256-4vrPoYIdfnoTCu6AiM355P8x0pI9P3V76bDMTRhJEv8=";
+    };
+
+  meta = with lib; {
+    description = "A color theme generator for editors and terminal emulators";
+    homepage = "https://github.com/benbusby/colorstorm";
+    license = licenses.mit;
+  };
```

```suggestion
    maintainers = with lib.maintainers; [ ];
  };
```

A placeholder if anyone wants to keep this in the future.